src/lib/core/core.gni

```diff
@@ -99,7 +99,8 @@ declare_args() {
   chip_tlv_validate_char_string_on_write = true
   chip_tlv_validate_char_string_on_read = false
 
-  chip_enable_sending_batch_commands = current_os == "linux"
+  chip_enable_sending_batch_commands =
+      current_os == "linux" || current_os == "mac" || current_os == "ios"
```

Comment:
Is `android` == linux?

Reply:
No that separate. I didn't have a way to test it, so I didn't feel like including it. I also didn't test mac or ios, but that was requested by Boris himself so I obliged